spec/api/comment_thread_spec.rb

```diff
@@ -565,7 +565,6 @@ def test_unicode_data(text)
 
       it "marks thread as read and confirms its value on returned response" do
         user = create_test_user(123)
-        thread = CommentThread.first
```

Comment:
we have a [helper method](https://github.com/edx-solutions/cs_comments_service/blob/ee9539c43f05de82fe299ada0b95f0900f35f530/spec/api/comment_thread_spec.rb#L453) at the start of this group which provides this variable for each test in this group, so no need of this variable declaration here